# An object, which has a static pose in the world, a type and a header

Header header 
string object_id
